### PR TITLE
Implement bootstrap modal for popup magazine

### PR DESCRIPTION
**Pull request by amritsiidhu**
Newletter is also downloadable on click and by button

### CODE REVIEW

**Comment by kenlt-uk** [comment]
Seems like the image used is low res, would be better to try and use a hi res image as it looks a bit blurry, can you do a prototype with this? 
low-res image for the thumbnails and hi-res image for preview of the magazine.

**Comment by amritsiidhu** [comment]
I can do that very simply but unfortunately i dont have any high res images. These images are what i was able to screenshot of the trello designs. This is also the same with the header logo and a few other assets which are all screenshots so therefore are low quality. Especially with the new colour scheme the assets had to be screenshotted from the preview image.

**Comment by kenlt-uk** [comment]
I would suggest going to newacropolisuk.org, click on the existing newsletter preview and taking a cropped screenshot of the hi res image. That's probably sufficient for now until I start working on newsletters properly.

**Comment by amritsiidhu** [comment]
I have now implemented a higher res image for the time being and improved the resizing of the modal

File: app/templates/views/home.html [hunk 1]
```diff
@@ -220,7 +220,29 @@ <h3>Read articles covering esoterica, philosophy, art and more…</h3>
           <p>Download the latest issue</p>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-4">
-          <img src="/static/images/home-image-magazine.png" class="img-fluid">
+          <a data-target="#modalIMG" data-toggle="modal" href="#" class="color-gray-darker td-hover-none">
+            <img class="card-img-top card_img_size" src="/static/images/home-image-magazine.png">
+            <div class="card-body">
+              <a data-target="#modalIMG" data-toggle="modal" href="#">View Image</a>
```
Comment (kenlt-uk) [inline app/templates/views/home.html]: Can you remove `View Image` link as I don't think its necessary.

Comment (AmritSidhu) [reply app/templates/views/home.html]: Done

Comment (amritsiidhu) [reply app/templates/views/home.html]: Done

**Pull request merged**